File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        StringBuilder str1 = new StringBuilder();
        StringBuilder str2 = new StringBuilder();
        
        for(int i = 0; i < word1.length; i++)
            str1.append(word1[i]);
        for(int i = 0; i < word2.length; i++)
            str2.append(word2[i]);
        return str1.toString().equals(str2.toString());
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.swift
class Solution {
    func arrayStringsAreEqual(_ word1: [String], _ word2: [String]) -> Bool {
        var strOne = ""
        var strTwo = ""
        
        for str in word1 {
            strOne += str
        }
        for str in word2 {
            strTwo += str
        }
        
        return (strOne == strTwo)
    }
}